src/model/ContentPageDef.ts

```diff
@@ -9,10 +7,9 @@ export type ColorPalette = '' | 'pc' | 'bc' | 'eo';
 export interface BaseBlockDef extends Slot {
   type?: string;
   content?: Record<string, any>;
-  version?: BlockVersion;
   hidden?: boolean;
   anchor?: string;
-  labels?: string[];
+  labels?: string[]; // TODO Remove and refactor group tabs
```

Comment:
```suggestion
  labels?: string[]; // TODO: Remove and refactor group tabs
```
Иначе `extensions` не смогут подсвечивать

Reply:
++